stable/concourse/templates/web-svc.yaml

```diff
@@ -13,8 +13,14 @@ spec:
     - name: atc
       port: {{ .Values.concourse.atcPort }}
       targetPort: atc
+      {{ if and (eq "NodePort" .Values.web.service.type) .Values.web.service.nodePort.fixed }}
```

Comment:
Nit but how about changing this to

```
{{ if and (eq "NodePort" .Values.web.service.type) .Values.web.service.atcNodePort }}`
nodePort: {{ .Values.web.service.atcNodePort}}
{{ end }}
```

so that:

*  We have one less setting key
   * `nodePort.fixed` becomes unnecessary
* The key naming looks more consistent
  * We already have `concourse.atcPort` for setting the port exposed by pods. IMHO `atcNodePort` looks more consistent with that.

Reply:
@mumoshu Your suggest is so good. Thanks!